haiku-apps/bephonebook/bephonebook-1.2.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A phone book manager"
+DESCRIPTION="BePhoneBook is a colorful phonebook made to help you stay \
+organized with your family's or friends' phone numbers."
+HOMEPAGE="https://github.com/HaikuArchives/BePhoneBook"
+COPYRIGHT="1999 Michael Melloff \
+	Fabien Fulhaber"
+LICENSE="BePhoneBook"
+REVISION="1"
+srcGitRev="69d6a5ada4fa76b0e900face8f6eef1d7ac48119"
```

Comment:
```diff
-srcGitRev="69d6a5ada4fa76b0e900face8f6eef1d7ac48119"
+rcGitRev="666d1cf3ea715f20455576664823c8ff85b4eec0
```
(Don't forget to drop the patchset and update `CHECKSUM_SHA256`)

Oops, you also need to add `haiku-apps/bephonebook/licenses/BePhoneBook`.